core-ui/src/components/Predefined/Create/OAuthClients/OAuthClients.create.js

```diff
@@ -188,5 +188,9 @@ OAuth2ClientsCreate.resourceGraphConfig = (t, context) => ({
       kind: 'Secret',
     },
   ],
+  matchers: {
+    OAuth2Client: (client, secret) =>
```

Comment:
```suggestion
    Secret: (client, secret) =>
```
We would match OAuth2Client with itself.